test/cypress/plugins/index.js

```diff
@@ -33,27 +33,35 @@ module.exports = (on, config) => {
      * the same call.
      */
     setupStubs (stubs) {
-      return request({
-        method: 'POST',
-        url: mountebankImpostersUrl,
-        json: true,
-        body: {
-          port: config.env.MOUNTEBANK_IMPOSTERS_PORT,
-          protocol: 'http',
-          defaultResponse: {
-            statusCode: 404,
-            body: 'No stub predicate matches the request',
-            headers: {}
-          },
-          stubs
-        }
+      return new Promise((resolve, reject) => {
+        axios.post(mountebankImpostersUrl,
```

Comment:
Is it possible to change this to await/async?
As we prefer using that syntax as per our Node style guide:
https://gds-way.cloudapps.digital/manuals/programming-languages/nodejs/#use-of-async-await

Reply:
Returning promise is fine in this case as this is expected by Cypress https://github.com/cypress-io/cypress/pull/6920/files

Reply:
Not sure if this is possible - but can we return a promise but change the axios code to use Async/Await?

E.g.

return new Promise(async (resolve, reject) => {
  try {
    // Axios code
    resolve()
  }catch (error){
    reject()
  }
})

Reply:
updated

Reply:
actually, it is not recommended to use async when returning promise https://eslint.org/docs/latest/rules/no-async-promise-executor

Reply:
I have removed returning Promise but it is not possible to change Axios usage to async/await without changing setupStubs to async

Reply:
Thanks for looking into that - I will approve